app/components/Button/button.css

```diff
@@ -20,6 +20,15 @@
   letter-spacing: -0.16px;
 }
 
+.button-secondary {
```

Comment:
how does this differ from the `.secondary` class below? That is conceptually what's the difference?

Reply:
The second version of the button, as represented in the figma, is significantly stylistically different, mainly as a result of the difference in hover states (the main button shows and outline, and the second shows a background). This also effects how sibling buttons are grouped. The `.secondary` class is a slight difference to the main button version, and the `.button-secondary` class is a second version of the button.